__tests__/src/components/common/SwitchLabeled.js

```diff
@@ -0,0 +1,23 @@
+import SwitchLabeled from '../../../../src/components/common/SwitchLabeled';
+import React from 'react';
+import renderer from 'react-test-renderer'
+import Strings from "../../../../src/global/Strings";
+
+test('SwitchLabeled renders correctly', () => {
```

Comment:
You can use [describe](https://facebook.github.io/jest/docs/en/api.html#describename-fn) method to compose tests for similar purpose (optional).